src/views/product-root-docs-path-landing/server.ts

```diff
@@ -73,70 +88,108 @@ const generateHeadingLevelsAndSidecarHeadings = ({
 	const sidecarHeadings = [
 		titleHeading,
 		...marketingContentHeadings,
-		...layoutHeadings.filter((heading) => heading.level !== 1),
+		...layoutHeadings.filter((heading: $TSFixMe) => heading.level !== 1),
 	]
 
 	return { sidecarHeadings, marketingContentBlocksWithHeadingLevels }
 }
 
-const generateGetStaticProps = ({
-	includeMDXSource = false,
-	pageContent,
+const generateRootDocsLandingPaths = () => {
+	return __config.dev_dot.beta_product_slugs.map(
+		(productSlug: ProductSlug) => ({
+			params: { productSlug },
+		})
+	)
+}
```

Comment:
🗒️ The paths are now dynamically based of off `__config.dev_dot.beta_product_slugs`.